pytest-selenium/pages/content.py

```diff
@@ -50,19 +106,33 @@ def click_toc_toggle_button(self):
             self.offscreen_click(self.toc_toggle_button)
             return self.page.sidebar.wait_for_region_to_display()
 
+
+
     class SideBar(Region):
         _root_locator = (By.CSS_SELECTOR, "[aria-label='Table of Contents']")
-
+        #_TOC_section2_locator = (By.XPATH, "//nav/ol/li[3]/nav/ol/li[1]/a")
+        _TOC_section2_locator = (By.CSS_SELECTOR, '[href$=to]')
```

Comment:
What is `_TOC_section2_locator`? is there not a more descriptive name. It's possible that there isn't.

Reply:
I am trying to click on a link in TOC. Here I am referring to section 2 link in TOC

Reply:
I have for now used toc_element. But this part of the script needs rework after the TOC layout is finalized. So I am not spending more time here